### ISSUE TITLE
Clarification about the loss

### ISSUE THREAD

**Issue by lparolari** [LeapLabTHU/Pseudo-Q#16]
> TL; DR
> 
> Can you explain what is loaded in the dataset along with image data? I would like to understand especially the content of `bbox`.

Dear authors,

I'm trying to figure out how the training of your model works. 

In particular, from this line https://github.com/LeapLabTHU/Pseudo-Q/blob/ce1688f7ffdfc20f624f9d8e0f3dc18a59ee5eea/engine.py#L40 I noticed that the `target` is used to compute the loss. The function `trans_vg_loss` confirms it: https://github.com/LeapLabTHU/Pseudo-Q/blob/ce1688f7ffdfc20f624f9d8e0f3dc18a59ee5eea/utils/loss_utils.py#L107-L125

I tried to understand what `target` is, and from this line https://github.com/LeapLabTHU/Pseudo-Q/blob/ce1688f7ffdfc20f624f9d8e0f3dc18a59ee5eea/engine.py#L30 I checked the `collate_fn` used in dataloader: https://github.com/LeapLabTHU/Pseudo-Q/blob/ce1688f7ffdfc20f624f9d8e0f3dc18a59ee5eea/utils/misc.py#L294-L308

Is this using the ground truth bounding box from the dataset?

I checked the `__getitem__` function from the dataset and I ended up with this three lines https://github.com/LeapLabTHU/Pseudo-Q/blob/ce1688f7ffdfc20f624f9d8e0f3dc18a59ee5eea/datasets/data_loader.py#L219-L221

Here a `.pth` file is loaded, and along with image data something else is loaded. Can you explain what the loaded `bbox` exactly contains?

Thank you,
Luca

**Comment by jianghaojun** [comment]
bbox contains (x, y, w, h)，(x, y) are bounding box's top-left corner coordinates and (w,h) are bounding box's width and height.
